components/composition/Hidden/Hidden.css

```diff
@@ -0,0 +1,50 @@
+/*
+  Prettier tried to put everything on new lines, but we don't want that in this file.
+  Can't see how to disable just that rule.
+*/
+/* stylelint-disable prettier/prettier */
+
+.diamond-hidden-small-mobile { display: none; }
+.diamond-visible-small-mobile { display: block; }
+
+@media (min-width: 375px) {
```

Comment:
I thought you said we should only use the new syntax now? `width >= 375px` or is that only because I had an instance of max-width in the button?

Reply:
Yeah we should probably make this global. I swear the linter flagged this to me the other day somewhere